app/assets/stylesheets/pages/_home.scss

```diff
@@ -1,4 +1,9 @@
 // Specific CSS for your home-page
+body {
+  background-image: url('https://images.pexels.com/photos/5778894/pexels-photo-5778894.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260');
```

Comment:
background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(.....);"

I copy pasted this from https://uikit.lewagon.com/documentation#banner